src/pages/index.tsx

```diff
@@ -60,15 +61,6 @@ const LabelInputContainer = ({
   );
 };
 
-const BottomGradient = () => {
```

Comment:
Why was this button specifically coded to have different color gradient from the rest of the form?